config/application.config.php

```diff
@@ -1,22 +1,29 @@
 <?php
-return [
-    // This should be an array of module namespaces used in the application.
-    'modules' => [
-        'ZendDeveloperTools',
-        'DoctrineModule',
-        'DoctrineORMModule',
-        'User',
-        'Application',
-        'Company',
-        'Decision',
-        'Frontpage',
-        'Activity',
-        'Photo',
-        'Education',
-        'Migration',
-        'MaglMarkdown'
-    ],
+$env = getenv('APP_ENV') ?: 'production';
```

Comment:
The zf2 docs suggest to do it this way... http://framework.zend.com/manual/current/en/tutorials/config.advanced.html

Of course following the zf2 docs is not necaserely the right thing to do here. However the differences between the production and development config are very minimal, so I'm doubting if it is worth the effort to maintain different config files for each.